tests/js/user-tests.js

```diff
@@ -371,6 +371,38 @@ module.exports = {
     });
   },
 
+  testAdminUser() {
+      return new Promise((resolve, reject) => {
+          if (!isNodeProcess) {
+              resolve();
+          }
+
+          // FIXME: check if adminUser() returns user iff valid token/server
+          let didThrow = false;
+          try {
+              let user = Realm.Sync.User.adminUser('THIS_IS_INVALID', 'http://localhost:9080');
+          }
+          catch (e) {
+              didThrow = true;
+              TestCase.assertTrue(e.message === 'Invalid adminToken or server.');
+          }
+          TestCase.assertTrue(didThrow);
+
+          // FIXME: find a way to set up a proper admin token
```

Comment:
we should skip checking the good path and rename the test method to indicate it tests the failure path only.